bitnami/concourse/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
-  version: 11.0.4
```

Comment:
Please remove this change from the PR since it belongs to concourse

Reply:
Done.Please review it again.Thank you^_^